### DISCUSSION TITLE
Explore @unitedstates GitHub for legal/legislative info

### DISCUSSION THREAD

**Discussion by nmrodelo** [PolicyEngine/policyengine-us#355] [General]
https://github.com/unitedstates?q=&type=&language=&sort=

**Comment by nikhilwoodruff** [comment]
[BillMap](https://billmap.linkedlegislation.com/) looks pretty useful!

**Reply by nmrodelo** [reply]
Yeah I wonder if we could use something like this to alert us that one of our policy parameters has changed, maybe auto generate an issue for us to investigate when a change goes through?
